packages/commcare/src/Adaptor.js

```diff
@@ -203,29 +172,41 @@ export function submit(formData) {
  * @returns {Operation}
  */
 export function fetchReportData(reportId, params, postUrl) {
-  return http.get(`api/v0.5/configurablereportdata/${reportId}/`, {
-    query: function (state) {
-      console.log(
-        'getting from url: '.concat(
-          state.configuration.baseUrl,
-          `/api/v0.5/configurablereportdata/${reportId}/`
-        )
-      );
-      console.log('with params: '.concat(params));
-      return params;
-    },
-    callback: function (state) {
-      var reportData = state.response.body;
-      return post(postUrl, {
-        body: reportData,
-      })(state).then(function (state) {
-        delete state.response;
+  return async state => {
+    const path = `/a/${state.configuration.applicationName}/api/v0.5/configurablereportdata/${reportId}/`;
+
+    console.log(
+      'getting from url: '.concat(
+        state.configuration.hostUrl,
+        `/a/${state.configuration.applicationName}/api/v0.5/configurablereportdata/${reportId}/`
+      )
+    );
+    console.log('with params: '.concat(params));
+
+    return await request({
```

Comment:
You should not mix `await` and `then`. Either use a promise chain (`then().then()`), or async/await.

Both styles are acceptable, just not at the same time :sweat_smile:

In this case you can just drop the `await` keyword and return the promise chain. That works nicely.